src/MissingTableHelper.php

```diff
@@ -12,7 +12,11 @@ public static function checkConfigs($configs, $dataDir)
         foreach ($configs as $config) {
             if ($config->getAttribute('mappings')) {
                 foreach ($config->getAttribute('mappings') as $name => $mapping) {
-                    $destinationBase = $dataDir . '/out/tables/' . $config->getAttribute('outputBucket') . '.';
+                    if ($config->getAttribute('outputBucket')) {
+                        $destinationBase = $dataDir . '/out/tables/' . $config->getAttribute('outputBucket') . '.';
+                    } else {
+                        $destinationBase = $dataDir . '/out/tables/';
+                    }
                     self::fillMissingTableMapping(
                         $destinationBase,
                         $config->getAttribute('outputBucket'),
```

Comment:
Ked ten `outputBucket` nie je definovany, tak co vrati tato metoda $config->getAttribute('outputBucket')? 
Jestli neudelat parameter `$outputBUcket` v metode `fillMissingTableMapping ` ako nepovinny (s default value).

Reply:
vrati false, no nevim co bych mu dal za default :)

Reply:
Njn, to by pri striktnom typovani nepreslo prave :) Tak to nechaj tak zatial.

Reply:
Ha v PHP 8 to uz zas pujde :D 
https://wiki.php.net/rfc/union_types_v2

Reply:
njnj, on by si ten generic uz zaslouzil nejakej komplexnejsi update - to jsou v tom jeste pozustatky extractor frameworku, tak to neni uplne cisty

Reply:
jasne, ak budeme pokracovat v takomto style ala Sporka, tak refaktoring mozes robit jedine o vikendech :trollface: